docs/scip/0004-data-loading.md

```diff
@@ -0,0 +1,92 @@
+# Template for new SCIPs
+
+## Metadata
+
+Editor:
+  ...
+
+Status (raw | draft | stable | deprecated | retired):
+  raw
+
+## Description
+
+...
+
+## Requirements
+
+### What is included in the catalog entry for a datasource?
+
+- A URL to a remote location (as given below)
+  - The URL should be a browsable location, structured according to one of the supported 'data-sharing patterns' (see below)
+  - 
+
+- An indication of additional scivision plugins required to load the data, if not (?)
+
+### Image formats
```

Comment:
what do you mean by "format"? 

Do you mean filetype like, `.jpg`, `.tif` etc?

Reply:
Yes exactly, will clarify